src/pages/List/List.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component, Fragment } from "react";
+import styled from "styled-components";
+import FilterWrap from "./FilterWrap";
+import { FILTERING } from "./FilteringComponent";
+import TicketSelctor from "../../Components/TicketSelector/TicketSelector";
+import TicketList from "./TicketList";
+import InfoContainer from "./InfoContainer";
+import PickedTicket from "./PickedTicket";
+import Navtrans from "../../Components/Navtrans/Navtrans";
+import LoadingScreen from "../../Components/LoadingScreen/LoadingScreen";
+import Selected from "./Selected";
+
+import { flexCenter, theme, flexColumn } from "../../styles/theme";
+
+class List extends Component {
+  constructor() {
+    super();
+    this.state = {
+      ticket: [],
+      selectOne: [],
+      loading: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://52.79.55.158:8000/products?limit=3")
```

Comment:
```suggestion
    fetch(`${API_FOO}/products?limit=3`)
```
이런 식으로 ip 주소 등은 src/config.js에 분리해주세요. 왜 이렇게 분리할까요?

Reply:
유지보수에 더 효과적이기 때문에 얼른가서 수정하겠습니다